apps/central-scan/backend/schema.sql

```diff
@@ -69,3 +69,9 @@ create table system_settings (
   id integer primary key check (id = 1),
   are_poll_worker_card_pins_enabled boolean not null default false
 );
+
+create table jurisdiction (
```

Comment:
Since this changes with the election configuration, could it just live in the same table?

Reply:
Updated! https://github.com/votingworks/vxsuite/pull/3348/commits/44c3e6fc8da181184fd8d4a8558bb49ad0419e96

This was one of those changes that ballooned a bit. In adding the jurisdiction to the election table and making it non-nullable, I had to update the store interface for setting the election to support simultaneously setting the jurisdiction. This required updating all consumers of `store.setElection` to use the new `store.setElectionAndJurisdiction`.

This updating led me to some unused handlers (unused as of https://github.com/votingworks/vxsuite/pull/3304), which instead of updating, I completely removed. I timeboxed this and left a TODO on two last handlers I didn't remove because removing them would require updates to trickier end-to-end tests.

Reply:
Thanks for doing that cleanup!